internal/db/db.go

```diff
@@ -196,6 +198,50 @@ func (db *db) AddPolicy(
 	return client.AddPolicyResult{PolicyID: policyID}, nil
 }
 
+func (db *db) AddDocActorRelationship(
+	ctx context.Context,
+	collectionName string,
+	docID string,
+	relation string,
+	targetActor string,
+) (bool, error) {
+	if !db.acp.HasValue() {
+		return false, client.ErrACPOperationButACPNotAvailable
+	}
+
+	if collectionName == "" {
+		return false, client.ErrOperationNotPermittedOnNamelessCols
```

Comment:
todo: This is already handled by `GetCollectionByName` which will error if no collection is found.  Please remove the new error and this block.

Reply:
good catch

- [x] removed